browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -140,6 +141,8 @@ WebUIController* NewWebUI(WebUI* web_ui, const GURL& url) {
       return new BravePrivateNewTabUI(web_ui, url.host());
     }
     return new BraveNewTabUI(web_ui, url.host());
+  } else if (host == kBraveGettingStartedHost) {
+    return new brave_education::BraveEducationUI(web_ui, url.host());
```

Comment:
It seems that upstream trend is to move webUIs to chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc and use WebUIConfig. We have a chromium_src override for it.